tests/test_material.py

```diff
@@ -1113,6 +1113,56 @@ def test_deepcopy():
     assert_equal(x, Material({'H1': 1.0}, mass=2.0, density=3.0, atoms_per_molecule=4.0,
                              metadata={'name': 'loki'}))
 
+
+def test_openmc():
+
+    leu = Material(nucvec={'U235': 0.04, 'U238': 0.96},
+                   metadata={'mat_number': 2,
+                          'table_ids': {'92235':'15c', '92238':'25c'},
+                          'mat_name':'LEU',
+                          'source':'Some URL',
+                          'comments': ('this is a long comment that will definitly '
+                                       'go over the 80 character limit, for science'),
+                          'name':'leu'},
+                   density=19.1)
+
+    mass = leu.openmc()
+    mass_exp = ('<material id="2" >\n'
+                '  <density value="19.10" units="g/cc" />\n'
+                '  <nuclide name="U235" wo="4.0000e-02" />\n'
+                '  <nuclide name="U238" wo="9.6000e-01" />\n'
+                '</material>\n')
+    assert_equal(mass, mass_exp)
+
+    atom = leu.openmc(frac_type='atom')
+    atom_exp = ('<material id="2" >\n'
+                '  <density value="19.10" units="g/cc" />\n'
+                '  <nuclide name="U235" ao="4.0491e-02" />\n'
+                '  <nuclide name="U238" ao="9.5951e-01" />\n'
+                '</material>\n')
+    assert_equal(atom, atom_exp)
+
+def test_openmc_mat0():
```

Comment:
I would make sure one of the test cases has an S(a,b) table too.

Reply:
Done. Caught an eminent formatting error along the way too.